.github/workflows/publish_image.yaml

```
@@ -23,6 +24,11 @@ jobs:
         # This is the 'default' image, it contains PostgreSQL 14 13 and 12, and defaults to PostgreSQL 14 binaries
         - pg_major: "14"
           pg_versions: "14 13 12"
+        # This is the 'default' image, along with Patroni changes to support static primaries
+        - pg_major: "14"
+          pg_versions: "14 13 12"
```

Nit, while we're evaluating this option, my guess is we'll only need PG14 binaries. That would reduce the image size and build time.

Unless we want to experiment with this feature on existing PG12 and PG13 deployments?
```suggestion
          pg_versions: "14 13 12"
```

---

That's fine by me. I don't have any reason to target 12 & 13. Will remove those.

---

Done.